### PR TITLE
Clean up JS linting

### PR DESCRIPTION
**Pull request by glennreyes**
Will solve #348 as well

### CODE REVIEW

**Comment by mxstbr** [comment]
This looks really good, thanks Glen! :fire:

File: server/logger.js [hunk 1]
```diff
@@ -12,26 +12,26 @@ const logger = {
...
 
     console.log(
-      chalk.bold('\nAccess URLs:') +
-      divider +
-      '\nLocalhost: ' + chalk.magenta('http://localhost:' + port) +
-      '\n      LAN: ' + chalk.magenta('http://' + ip.address() + ':' + port) +
-      (tunnelStarted ? '\n    Proxy: ' + chalk.magenta(tunnelStarted) : '') +
-      divider,
-      chalk.blue('\nPress ' + chalk.italic('CTRL-C') + ' to stop\n')
+      chalk.bold(`
```
Comment (Deveritas) [inline server/logger.js]: Won't this bold the whole string now instead of just `Access URLs:`? Or am I missing something?

Comment (mxstbr) [inline server/logger.js]: Yeah, good catch! I haven't tried it, but that sounds like a likely error–does that happen @glennreyes?

Comment (glennreyes) [inline server/logger.js]: Yeah that was dumb, I just commited a fix of that! Thanks @Deveritas 🙌

**Pull request merged**